Gandheesh/1.Basics.html

```diff
@@ -0,0 +1,51 @@
+<html>
+     <head><meta charset="UTF-8">
```

Comment:
Indent opening and closing of head tag one below another.
In VS Code press ctrl + shift + P and type format document in the text box, this will indent your code automatically.